apps/mobile/src/store/settings/settings.ts

```diff
@@ -111,6 +117,9 @@ export function useSettings() {
     changeThemePreference(theme: ThemePreference) {
       dispatch(userChangedThemePreference(theme));
     },
+    changeLastUnlocked(timestamp: LastUnlockedTimestamp) {
+      dispatch(userChangedLastUnlocked(timestamp));
+    },
```

Comment:
[Modelling after a user event](https://redux.js.org/style-guide/#model-actions-as-events-not-setters), this would be along the lines of `userUnlocksWallet`, and the result that happening is that we update the last unlocked time

Reply:
Yep that sounds good